includes/class-used-css.php

```
@@ -0,0 +1,1206 @@
+
+		/**
+		 * Get the used-CSS cache file path for a URL.
+		 *
+		 * @param string $url The page URL.
+		 * @return string The filesystem path.
+		 * @since 1.9.0
+		 */
+		public function get_used_css_path( string $url = '' ): string {
+			$path = $this->get_url_path( $url );
+			return "{$this->cache_root_dir}/{$this->domain}/{$path}/" . self::USED_CSS_FILENAME;
+		}
```

PHPMD (2.15.0)

[error] The method get_used_css_path is not named in camelCase. (undefined)

(CamelCaseMethodName)